### PR TITLE
CI/CD updates

### PR DESCRIPTION
**Pull request by tonymorony**

### CODE REVIEW

**Comment by Milerius** [comment]
All checked passed, if @tonymorony consider it as ready, i'm ok for merging it

**Comment by tonymorony** [comment]
> All checked passed, if @tonymorony consider it as ready, i'm ok for merging it

Sure, lets merge it. I'll deliver more updates next week

File: .github/workflows/atomicdexpro_ci.yml [hunk 1]
```diff
@@ -249,8 +276,40 @@ jobs:
...
+          scoop cache rm nim
+
+          $file = 'src\generate.nim'
+          $regex = '(?<=g_vcpkg_cmake_script_path & ")[^"]*'
+          (Get-Content $file) -replace $regex, ' -DVCPKG_TARGET_TRIPLET=x64-windows ' | Set-Content $file
+          nimble build -y
+          $Env:QT_INSTALL_CMAKE_PATH = "C:\Qt\5.14.2\msvc2017_64"
+          $SHAORIG = "A31827929A4D73EB71BD6986C38EB16D4E779DC3880EFA520869D00C3C0E2AB9"
+          $DWFILE = ($PWD | select -exp Path) + '\debuglibs-win-dexpro.zip'
+          (New-Object System.Net.WebClient).DownloadFile('https://sirseven.me/share/debuglibs-win-atomicdexpro.zip', $DWFILE)
```
Comment (tonymorony) [inline .github/workflows/atomicdexpro_ci.yml:308]: TODO:  reminder, Have to fix this one before undraft

**Comment by ca333** [comment]
> Sure, lets merge it. I'll deliver more updates next week

fyi - this is still in draft.

**Comment by tonymorony** [comment]
> > Sure, lets merge it. I'll deliver more updates next week
> 
> fyi - this is still in draft.

Yep, I've switched it back to draft because noticed https://github.com/KomodoPlatform/atomicDEX-Pro/pull/152#pullrequestreview-412781924 on re-read

File: ci_tools_atomic_dex/src/dependencies.nim [hunk 2]
```diff
@@ -2,6 +2,7 @@ import osproc
 import vcpkg
 
 let g_packages = [
+    (name: "jsoncpp", head: false),
```
Comment (Milerius) [inline ci_tools_atomic_dex/src/dependencies.nim:5]: This is spurious, the library for nlohmann json in vcpkg is https://github.com/microsoft/vcpkg/tree/master/ports/nlohmann-json

**Pull request merged**